recipe/meta.yaml

```diff
@@ -47,14 +61,15 @@ test:
     - papermill --help
 
 about:
-  home: https://github.com/nteract/papermill
-  license_file: LICENSE
+  home: http://github.com/nteract/papermill
   license: BSD 3-Clause
   license_family: BSD
-  summary: 'Papermill is a tool for parameterizing, executing, and analyzing Jupyter Notebooks.'
-  doc_url: http://papermill.readthedocs.io/en/latest/
+  license_file: LICENSE
+  summary: "Papermill parameterizes, executes, and analyzes Jupyter Notebooks."
+  doc_url: http://papermill.readthedocs.io/
   dev_url: https://github.com/nteract/papermill
 
 extra:
   recipe-maintainers:
-    - pmlandwehr
```

Comment:
Why remove @pmlandwehr ? He can help you with maintaining this feedstock.

Reply:
I'm fine with adding this back. It seemed unmaintained so wasn't sure if they wanted to maintain.

Reply:
Depending on the response to https://github.com/conda-forge/papermill-feedstock/issues/12, I can update this PR if needed.